relay-server/src/utils/rate_limits.rs

```diff
@@ -466,40 +508,7 @@ where
     /// - Rate limits declare all active rate limits, regardless of whether they have been applied
     ///   to items in the envelope. This excludes rate limits applied to required attachments, since
     ///   clients are allowed to continue sending them.
-    ///
-    /// # Example
```

Comment:
Can we keep these docs, they seem to be still relevant? We just don't drop the items anymore directly.

Reply:
I wanted to rewrite them in the new function. Will do in a follow up.